### DISCUSSION TITLE
Add "Hide target" to entity hider

### DISCUSSION THREAD

**Discussion by BopOSRS** [runelite/runelite#17788] [Ideas]
Already have hide attackers, why not other way around..

**Comment by Alexsuperfly** [comment]
What purpose does hiding the thing you're trying to interact with have

**Reply by BopOSRS** [reply]
Can see under or through whatever you're attacking etc.
Like if core spawns on corp can see it behind/under without having to right click or use npc indicator shenanigans
probs also for like sotetseg or stuff where you could already hide it if you wanted to see the projectiles under boss

**Reply by DominickCobb-rs** [reply]
The logic you're expecting to have the thing you're attacking isn't unhidden by the time you're ready to attack again isn't there.

**Reply by BopOSRS** [reply]
I don't know what you've just tried to say
